packages/Benchmarks/HybridTransport/src/DTK_Benchmark_CartesianMesh.hpp

```diff
@@ -75,11 +75,6 @@ namespace Benchmark
 class CartesianMesh
 {
   public:
-    /*!
-     * \brief Default constructor.
-     */
-    CartesianMesh() { /* ... */}
```

Comment:
I switched the `CartesianMesh` to be a standalone data structure and so only wanted to be able to create on at the time of initialization instead of having a separate build function that could be called after construction.